native/ssz_nif/src/elx_types/p2p.rs

```diff
@@ -34,6 +34,22 @@ gen_struct_with_config!(
     }
 );
 
+gen_struct!(
+    #[derive(NifStruct)]
+    #[module = "SszTypes.BeaconBlocksByRootRequest"]
+    pub(crate) struct BeaconBlocksByRootRequest<'a> {
+        block_roots: Vec<Root<'a>>,
+    }
+);
+
+gen_struct_with_config!(
+    #[derive(NifStruct)]
+    #[module = "SszTypes.BeaconBlocksByRootResponse"]
+    pub(crate) struct BeaconBlocksByRootResponse<'a> {
+        blocks: Vec<SignedBeaconBlock<'a>>,
```

Comment:
We can't really use these response types, because each encoded block is prepended with a header, and sent in a different chunk, so we can remove the `BeaconBlocksByRootResponse` implementation.